Script/AtomicNET/AtomicNET/Resource/ResourceCache.cs

```diff
@@ -29,6 +30,20 @@ public System.IO.Stream GetFileStream(string name, bool sendEventOnFailure = tru
             return null;
         }
 
+        public List<string> GetResourceDirs()
```

Comment:
Please add a basic comment summary for the method, see the ```ReleaseAllResources``` method immediately following  for an example.  Choosing the container return type is important,  Array, Vector, and List are all choices here.  Array is the best one as the return value isn't intended to be modified with additions/subtractions (Vector), or randomly inserted into (List)